tests/NHSD.BuyingCatalogue.Solutions.Application.UnitTests/Solutions/SolutionGetByIdTests.cs

```diff
@@ -133,7 +134,16 @@ public async Task ShouldGetSolutionById()
             solution.Hosting.PublicCloud.RequiresHSCN.Should().Be("It is required");
 
             solution.IsFoundation.Should().BeTrue();
-            solution.Capabilities.Should().BeEquivalentTo(new[] {"cap1", "cap2"});
+            solution.Capabilities.Should().HaveCount(2);
+            solution.Capabilities.Should()
+                .BeEquivalentTo(
+                    new[]
+                    {
+                        Mock.Of<IClaimedCapability>(cc => cc.Name == "cap1"),
+                        Mock.Of<IClaimedCapability>(cc => cc.Name == "cap2")
+                    }, config => config.ComparingByMembers<IClaimedCapability>());
```

Comment:
Not seen this before, the config looks interesting. What does it do?

Reply:
I think this is the first time we've compared an array of complex objects in a test. This tells it to do a memberwise comparison of the two objects (otherwise we get a reference comparison), so I can compare the mocked version of IClaimedCapability with the returned value (which will be ClaimedCapabilityDto under the hood)